packages/tasks/src/tasks/audio-classification/data.ts

```diff
@@ -70,7 +74,7 @@ const taskData: TaskDataCustom = {
 	],
 	summary:
 		"Audio classification is the task of assigning a label or class to a given audio. It can be used for recognizing which command a user is giving or the emotion of a statement, as well as identifying a speaker.",
-	widgetModels: ["facebook/mms-lid-126"],
+	widgetModels: ["speechbrain/lang-id-voxlingua107-ecapa"],
```

Comment:
It'd be better to highlight Audio Spectrogram Transformer here: `MIT/ast-finetuned-audioset-10-10-0.4593`

https://huggingface.co/MIT/ast-finetuned-audioset-10-10-0.4593